### DISCUSSION TITLE
Proxy Disabled?

### DISCUSSION THREAD

**Discussion by winkmichael** [Ysurac/openmptcprouter#2400] [Q&A] [answered]
Hello all,

On my OpenMPTCP Status page, I always get an error "Proxy DISABLED." I've tried switching between proxy types via the Wizard with no luck. Can anyone point me in the right direction to fix this?

Thanks

**Comment by 7router7** [comment]
I have the same issue, using latest server and client versions.
Did you solve the problem?

**Comment by winkmichael** [comment, marked as answer]
Using the latest August release, the fix for me was to use static IP addresses. DHCP worked for one interface to get an address, but everything send suck. The second interface would never get an address. Once i switched to static IPs everything came together perfect.
